src/Type/Type.php

```diff
@@ -78,7 +78,7 @@
      *
      * @return string
      */
-    public function generateForm( string $language = 'en', $post = null) : string
+    public function generateForm(string $language = 'en', $post = null) : string
```

Comment:
__[phpmd]__
* The method generateForm() has a Cyclomatic Complexity of 14. The configured cyclomatic complexity threshold is 10.

<!-- Comment by [SideCI](https://sideci.com) -->